interface/wx/grid.h

```diff
@@ -2945,7 +2945,14 @@ class wxGrid : public wxScrolledCanvas
 
             @since 2.9.1
          */
-        wxGridSelectRowsOrColumns
+        wxGridSelectRowsOrColumns,
+
+        /**
+            The selection mode allowing no selections to be made at all.
+
+            The user won't be able to select any cells in this mode.
```

Comment:
Please add a `@since 3.1.5` line to this comment (similar to the comment above it).

Reply:
Done! Sorry about that - I wasn't sure if I was meant to add it myself or not so just left it!